src/dropdown.js

```diff
@@ -278,6 +285,15 @@ const styles = {
         boxSizing: 'border-box',
         outline: 'none',
     },
+    dropdownFooter: {
+        padding: '0.5em',
+        position: 'sticky',
```

Comment:
does position sticky work when the list is short? Flexbox may be more reliable in this case.